src/components/dialogs/create-case-dialog/create-case-dialog.tsx

```diff
@@ -180,24 +171,19 @@ const CreateCaseDialog: React.FunctionComponent<ICreateCaseDialogProps> = ({
         }
     );
 
-    useEffect(() => {
-        setIsCreationAllowed(
+    const isCreationAllowed = useMemo(
```

Comment:
With a bit of effort, you can delete this method and use only yup validation instead.

Reply:
yes it is better.
done, thank you